mypy/waiter.py

```diff
@@ -161,12 +189,14 @@ def _record_time(self, name: str, elapsed_time: float) -> None:
 
     def _poll_current(self) -> Tuple[int, int]:
         while True:
-            time.sleep(.05)
+            time.sleep(.01)
```

Comment:
Performance improvements by trimming the [slowdown loops](http://thedailywtf.com/articles/The-Slow-Down-Loop)!